scripts/Package_BGA/csp.yml

```diff
@@ -12,7 +12,7 @@ ST_WLCSP-25_Die444:
   pkg_height: 2.325
   pitch: 0.4
   pad_diameter: 0.225
-  paste_diameter: 0.25
+  paste_margin: 0.25
```

Comment:
`paste_margin` isn't simply a new name for `paste_diameter` - what you've done here makes paste openings that extend 0.25 mm beyond the edges of the pads, but they should only extend 0.0125 mm beyond the edges of the pad according to the datasheet.